src/Game.py

```diff
@@ -14,16 +14,26 @@ def __init__(self):
 
     def runGame(self):
         pygame.init()
+        clock = pygame.time.Clock()
         pygame.display.set_caption('Basic Pygame program')
         self.isRunning = True
         while self.isRunning:
+            dt = clock.tick(clock.get_fps())/1000
```

Comment:
Why are we using frameRate here?
I don't think we need frameRate, we can simply use a predefined propagate speed of 0.5 pixels per frame or so for now atleast